{
  "Header": {
    "Original": "CVE-2021-44476 (A sandboxing issue in Odoo Community 15.0 and earlier and Odoo Enterpr ...)",
    "Line": 169980,
    "ID": "CVE-2021-44476",
    "Description": "(A sandboxing issue in Odoo Community 15.0 and earlier and Odoo Enterpr ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-5399-1}",
      "Line": 169981,
      "Type": "xref",
      "Bugs": [
        "DSA-5399-1"
      ]
    },
    {
      "Original": "- odoo 16.0.0+dfsg.1-1 (bug #1035953)",
      "Line": 169982,
      "Type": "package",
      "Package": "odoo",
      "Kind": "fixed",
      "Version": "16.0.0+dfsg.1-1",
      "BugNo": 1035953
    },
    {
      "Original": "NOTE: https://github.com/odoo/odoo/issues/107684",
      "Line": 169983,
      "Type": "NOTE",
      "Description": "https://github.com/odoo/odoo/issues/107684"
    },
    {
      "Original": "NOTE: 14.0 patch at https://github.com/odoo/odoo/commit/be2c857a2e19b0a752555ab377ce5e1cb081a186",
      "Line": 169984,
      "Type": "NOTE",
      "Description": "14.0 patch at https://github.com/odoo/odoo/commit/be2c857a2e19b0a752555ab377ce5e1cb081a186"
    }
  ]
}